### PR TITLE
Reducers and actions

### PR DESCRIPTION
**Pull request by hammaazarok**
## in this milestone I achieved the following :
-  Create redux reducer actions and action types for Books
-  Create redux reducer actions and action types for categories
-  Create a redux store and connected it to react app

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Actions
+const LOAD = 'LOAD';
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```
Comment (omar-muhamad) [inline src/redux/books/books.js]: - [x] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), So please edit all action types to be something like that:
`const ADD_BOOK = bookstore/books/ADD_BOOK`

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const CHECK = 'CHECK';
+
```
Comment (omar-muhamad) [inline src/redux/categories/categories.js]: - [x] Same here, Please edit this action type to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @hammaazarok,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by hammaazarok** [comment]
@Omar-Muhamad thanks for the helpful review changes applied.

**Review by sumairq — APPROVED** [review]
### Hi @hammaazarok  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**